.github/ISSUE_TEMPLATE/template-bug.yml

```diff
@@ -0,0 +1,179 @@
+name: Bug Report
+description: A generic bug report
+title: "[BUG]: "
+labels: ["Type: Bug", "Status: Available"]
```

Comment:
Will it work with new labels? 🤨

Reply:
nope, it should be fixed by me in few days)